normandy/recipes/filters.py

```diff
@@ -636,6 +636,37 @@ def capabilities(self):
         return set()
 
 
+class NegateFilter(BaseFilter):
+    """
+    This filter takes the JSON needed for a filter as an
+    argument,  and this filter will "negate" it.
+
+
+    .. attribute:: type
+
+        ``negate``
+
+    .. attribute:: data_for_filter
+
+        This is a JSON object that would be passed to the filter
+        you want to negate.
+
+       :example: `{ "type": "channel", "channels": ["release", "beta"]}`
```

Comment:
These docs don't match the actual arguments passed. For what it's worth, I prefer `filter_to_negate` over `data_for_filter`.

The phrasing "that would be passed to the filter" is a bit odd too, since the object is a filter, not something that is passed to a filter. Filters aren't really functions or things that take in data. They are descriptions of behavior.